pkg/generated/openapi/cmd/models-schema/main.go

```diff
@@ -19,12 +19,11 @@ package main
 import (
 	"encoding/json"
 	"fmt"
-	"os"
-	"strings"
-
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/kube-openapi/pkg/common"
 	"k8s.io/kube-openapi/pkg/validation/spec"
 	"k8s.io/kubernetes/pkg/generated/openapi"
+	"os"
```

Comment:
nit , usually the stdlib imports go first and separated

Reply:
gah, my IDE's import settings got reset. Fixing..